Tests/AutoTests/FunctionalTest/functional.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { test, expect, chromium, Page, Browser, BrowserContext } from '@playwright/test';
+import * as app from "../../Configuration/appConfig.json";
+import home from '../../pageObject/home.page';
+
+test.describe("Goto maritime-safety-information Home Page", ()=> {
```

Comment:
Can you please change this test template to be like accessebility.spec.ts ?

Reply:
the functional spec file template is resolved as accessebility spec file